app/templates/components/speaker-item.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="speaker-item">
+  {{#if speaker.image}}
+    <img alt="" class="ui small picture" src="{{speaker.image}}">
+  {{else}}
+    <img alt="" class="ui small picture" src="/images/placeholders/avatar.png">
+  {{/if}}
+  <br>
+  <div>{{ speaker.name }}</div>
+  <div>{{ speaker.organisation }}</div>
+  <div class="social-links">
+    {{#each speaker.links as |links|}}
```

Comment:
should be `{{#each speaker.links as |link|}}`. Use `link` instead of `links`. Each item should be denoted by a singular name.